src/allotropy/parsers/appbio_quantstudio/appbio_quantstudio_structure.py

```diff
@@ -118,6 +118,11 @@ class WellItem(Referenceable):
     well_location_identifier: str | None = None
     quencher_dye_setting: str | None = None
     sample_role_type: str | None = None
+    sample_color: str | None = None
+    biogroup_name: str | None = None
+    biogroup_color: str | None = None
+    target_name: str | None = None
```

Comment:
@nathan-stender, we should already have mapping for `target name` as ASM term `target DNA description`. Probably can disregard adding that field. Rest of the changes LGTM!

Reply:
Oh good call, done!